src/tests/test_dte_data_models.py

```diff
@@ -820,19 +892,27 @@ def test_init_fail_ind_global_invalid(self) -> None:
 
         obj = self.obj_1
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 obj,
                 ind_global=2,
             )
         self.assertEqual(
-            assert_raises_cm.exception.errors(),
+            assert_raises_cm.exception.errors(
+                include_context=False,
+                include_input=False,
+                include_url=False,
+            ),
             [
                 {
                     'loc': ('ind_global',),
-                    'msg': '("Only the value \'1\' is valid for the field \'ind_global\'", 2)',
+                    'msg': (
+                        'Value error, ('
+                        '"Only the value \'1\' is valid for the field \'ind_global\'", 2'
+                        ')'
+                    ),
                     'type': 'value_error',
-                }
+                },
```

Comment:
Revert change to reduce diff size (unless the comma was added by Black).

```suggestion
                }
```